usage.md

```diff
@@ -64,17 +64,17 @@ units: notaurl
 # gather, print to stdout
 
 ```
-$ irule -r irods_rule_engine_plugin-irods_rule_language-instance "metadata_templates_collection_gather('*logical_path', '*recursive', *schemas); writeLine('stdout', *schemas)" '*logical_path=/tempZone/home/rods/thedir%*recursive=0%*schemas=""' ruleExecOut
+$ irule -r irods_rule_engine_plugin-irods_rule_language-instance "metadata_templates_collection_gather('*logical_path', '*recursive', *schemas); writeLine('stdout', *schemas)" '*logical_path=/tempZone/home/rods/thedir%*recursive=0%*schemas=' ruleExecOut
```

Comment:
You could shorten these lines by hard-coding the inputs directly in the rule call.

With that said, I'm guessing the use of the INPUT list is to make the rule code logic easier to understand?

Reply:
as command line examples, they were easier to change... and yes, I think they are more instructive as INPUTs to any reader...

of course, the usage.md file probably just becomes a README section in the PREP once everything is merged.